arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1592,7 +1588,8 @@ void ClusterInfo::loadPlan() {
               auto it = newShardGroups.find(groupLeaderCol->second->at(i));
               if (it == newShardGroups.end()) {
                 // Need to create a new list:
-                auto list = std::make_shared<std::vector<ShardID>>(2);
+                auto list = std::make_shared<std::vector<ShardID>>();
+                list->reserve(2);
                 // group leader as well as member:
                 list->emplace_back(groupLeaderCol->second->at(i));
                 list->emplace_back(col->second->at(i));
```

Comment:
This looks inconsistent now. We are reserving space for 2 elements, but are adding 3.
And the 2nd and 3rd elements look identical. I don't think this is how it should be.

Reply:
Sorry, as far as I see this puts exactly two elements in the list, one is the group leader, the other is one group member. Later, we might get more, but this is then in another round. I think this is correct.